Tabs.php

```diff
@@ -240,8 +244,11 @@ protected function renderDropdown($itemNumber, &$items, &$panes)
             if (isset($item['visible']) && !$item['visible']) {
                 continue;
             }
-            if (!array_key_exists('content', $item)) {
-                throw new InvalidConfigException("The 'content' option is required.");
+            if (!(array_key_exists('content', $item) xor array_key_exists('url', $item))) {
```

Comment:
I considered that but given that a URL will prevent the content from ever showing up I don't think it is a good idea to actually allow both at the same time from a usability/expected behavior perspective. Using `xor` makes sure that the output is consistent and predictable.